src/app/services/actions/permissions-action-creator.ts

```diff
@@ -43,8 +47,9 @@ export function fetchScopes(): Function {
   return async (dispatch: Function, getState: Function) => {
     let hasUrl = false; // whether permissions are for a specific url
     try {
-      const { devxApi, permissionsPanelOpen, sampleQuery: query }: IRootState = getState();
+      const { devxApi, permissionsPanelOpen, permissionModeType, sampleQuery: query }: IRootState = getState();
       let permissionsUrl = `${devxApi.baseUrl}/permissions`;
+      const scope = permissionModeType === DISPLAY_DELEGATED_PERMISSIONS ? PERMS_SCOPE.WORK : PERMS_SCOPE.APPLICATION;
```

Comment:
```suggestion
      const scope = permissionModeType === (DISPLAY_DELEGATED_PERMISSIONS 
          ? PERMS_SCOPE.WORK 
          : PERMS_SCOPE.APPLICATION);
```
This line feels a bit weird without a linebreak, I think